internal/cli/root.go

```diff
@@ -93,6 +103,15 @@ func (c *Client) apiConnect(httpClient *http.Client) error {
 	return nil
 }
 
+func (c *Client) metalApiConnect(httpClient *http.Client) error {
+	configuration := metal.NewConfiguration()
+	configuration.Debug = checkEnvForDebug()
+	configuration.AddDefaultHeader("X-Auth-Token", c.Token())
+	metalgoClient := metal.NewAPIClient(configuration)
```

Comment:
We'll want to put the UserAgent here, as you did in #276 
https://github.com/equinix/metal-cli/pull/276/files#diff-aa84670cfbda5699ea29a1dad10df2df47f189c2748c57f6ea3efff057313658R110.

Reply:
Also covered in #274